ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/builder/BuilderBlockCreator.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
Same here, don't add licence & copyrights in the files you wrote.
Please apply to the other new files as well